content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,398 @@
+---
+title: Deployment Models
+description: Describes the system models that impact your overall Istio depolyment.
+weight: 60
+keywords:
+- single-cluster
+- multiple-clusters
+- control-plane
+- tenancy
+- networks
+- identity
+- trust
+- single-mesh
+- multiple-meshes
+aliases:
+- /docs/concepts/multicluster-deployments/
+---
+
+Important system models impact your overall Istio deployment model. This page
+discusses the options for each of these models and describes how you can
+configure Istio to address them.
+
+## Cluster models
+
+A cluster is a set of compute nodes that run containerized applications.
+Typically, the compute nodes comprising a cluster can reach each other directly
+and you can limit external access through rules or policies. For isolation,
```

Comment:
The first two sentences are the same text as the glossary, so would it make more sense to replace the them with something like:

```
Workloads of an Istio application run in one or more {{< gloss >}}clusters{{< /gloss >}}.
```

Reply:
Agreed, I am using the introduced terminology though:
```
The workload instances of your application run in one or more {{< gloss "cluster" >}}clusters{{< /gloss >}}. For ...
```